examples/0000-test-config-small-network.yaml

```diff
@@ -10,7 +10,7 @@ relaychain:
     - name: alice
       validator: true
     - name: bob
-      image: 'docker.io/paritypr/polkadot-debug:5236-0.9.18-c55660e9-be16bd72'
+      image: 'docker.io/parity/polkadot:latest'
```

Comment:
I think the idea of this one was show that you can define a `default_image` to use in the nodes and override the `image` in one. The goal is running a network with different versions.

Reply:
The idea of the very first example 0000 (in my head at least) is to show the different config files that can be used. What you mention will be applied in one of the next ones - but thank you very much for pointing it out!

Reply:
Added as part of 0002 test (different images)